lms/djangoapps/ccx/utils.py

```diff
@@ -170,7 +170,8 @@ def get_valid_student_with_email(identifier):
         identifier (str): Username or email of the user to enroll
 
     Returns:
-        str: A validated email for the user to enroll
+        str: A validated email for the user to enroll.
```

Comment:
The return type here should be a tuple. Can you update the documentation accordingly? For example, see the accepted answer to [this question](http://stackoverflow.com/questions/29221551/can-sphinx-napoleon-document-function-returning-multiple-arguments), the Napoleon Google style